CRM/Utils/Check/Component/Env.php

```diff
@@ -986,4 +986,18 @@ public function checkMysqlVersion() {
     return $messages;
   }
 
+  public function checkPHPIntlExists() {
+    $messages = [];
+    if (!extension_loaded('intl')) {
+      $messages[] = CRM_Utils_Check_Message(
```

Comment:
Should be `$messages[] = new CRM_Utils_Check_Message(`. It gives an error 500 currently.